packages/core/src/generators/angular/index.ts

```diff
@@ -118,6 +118,167 @@ const BINDINGS_MAPPER: { [key: string]: string | undefined } = {
   style: 'ngStyle',
 };
 
+const handleObjectBindings = (code: string) => {
+  let objectCode = code.replace(/^{/, '').replace(/}$/, '');
+  objectCode = objectCode.replace(/\/\/.*\n/g, '');
+
+  const spreadOutObjects = objectCode
+    .split(',')
+    .filter((item) => item.includes('...'))
+    .map((item) => item.replace('...', '').trim());
+
+  const objectKeys = objectCode
+    .split(',')
+    .filter((item) => !item.includes('...'))
+    .map((item) => item.trim());
+
+  const otherObjs = objectKeys.map((item) => {
+    return `{ ${item} }`;
+  });
+
+  let temp = `${spreadOutObjects.join(', ')}, ${otherObjs.join(', ')}`;
+
+  if (temp.endsWith(', ')) {
+    temp = temp.slice(0, -2);
+  }
+
+  if (temp.startsWith(', ')) {
+    temp = temp.slice(2);
+  }
+
+  // handle template strings
+  if (temp.includes('`')) {
+    // template str
+    let str = temp.match(/`[^`]*`/g);
+
+    let values = str && str[0].match(/\${[^}]*}/g);
+    let forValues = values?.map((val) => val.slice(2, -1)).join(' + ');
+
+    if (str && forValues) {
+      temp = temp.replace(str[0], forValues);
+    }
+  }
+
+  return temp;
+};
+
+const processCodeBlockInTemplate = (code: string) => {
+  // contains helper calls as Angular doesn't support JS expressions in templates
+  if (code.startsWith('{')) {
+    // Objects cannot be spread out directly in Angular so we need to use `useObjectWrapper`
+    return `"useObjectWrapper(${handleObjectBindings(code)})" `;
+  } else if (code.startsWith('Object.values')) {
+    let stripped = code.replace('Object.values', '');
+    return `"useObjectDotValues${stripped}" `;
```

Comment:
This is missing parentheses no?
```suggestion
    return `"useObjectDotValues( ${stripped} )" `;
```

Reply:
OH nevermind, the parentheses are included in `stripped`, since it was an `Object.values(x)` call before.